CMakeLists.txt

```diff
@@ -28,5 +28,8 @@ file(GLOB SOURCE_FILES "src/*.cpp" "src/*.h")
 # Declare target executable
 add_executable(${TARGET} ${SOURCE_FILES})
 
+# Link libraries
+target_link_libraries(${TARGET} template-library)
```

Comment:
I think we should add the whole example with the .pc file, otherwise it's a bit weird that you'd need to install it into your system just to get it to compile.